Fibonacci.c

```diff
@@ -0,0 +1,55 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Ayisha Sameera,GET.
+// ------------------------------------------------------------------
+// Test.c
+// Program on branch AB2.
+// ------------------------------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+
+static void Print (int i, int num) {
+   printf ("Term %-3d:  %-5d\n", i, num);
+}
+
+static void PrintFibonacci (int num) {
+   int n1 = 0;
+   int n2 = 1;
+   if (num > 0) Print (1, 0);
+   if (num > 1) Print (2, 1);

```

Comment:
You do not need this comparison. If num < 0 it is invalid and you should return. Otherwise your for loop will handle it.